src/Braket.jl

```diff
@@ -134,6 +139,72 @@ end
 Base.copy(fp::FreeParameter) = fp
 Base.show(io::IO, fp::FreeParameter) = print(io, string(fp.name))
 
+"""
+    FreeParameterExpression
+    FreeParameterExpression(expr::Union{FreeParameterExpression, Number, Symbolics.Num, String})
+
+Struct representing a Free Parameter expression, which can be used in symbolic computations.
+
+### Examples
+```jldoctest
+julia> fp_alpha = FreeParameter(:alpha)
+alpha
+
+julia> fp_beta = FreeParameter(:beta)
+beta
+
+julia> expr1 = FreeParameterExpression("2 * alpha / 3")
+(2//3)*alpha
+
+julia> expr2 = FreeParameterExpression("alpha + 2 * beta")
+alpha + 2beta
+```
+"""
+
+struct FreeParameterExpression
+    expression::Symbolics.Num
+
+    function FreeParameterExpression(expr::Union{FreeParameterExpression, Number, Symbolics.Num, String})
```

Comment:
Why not do this in a Julia-idomatic way and use dispatch?

Reply:
Indeed! That's makes better sense! It's done!